tcadmin/tests/conftest.py

```diff
@@ -19,3 +19,12 @@ def pytest_collection_modifyitems(config, items):
         for item in items:
             if "slow" in item.keywords:
                 item.add_marker(skip_slow)
+
+
+@pytest.fixture(scope="module")
+def appconfig():
+    from tcadmin.appconfig import AppConfig
```

Comment:
Please include the comment here as well!

Reply:
Fixed